e2e/testcases/correction-birth/correct-birth-record-6.spec.ts

```diff
@@ -300,17 +305,14 @@ test.describe.serial(' Correct record - 6', () => {
 
       await expect(
         page.locator('#informant-content #Date').getByRole('deletion')
-      ).toHaveText(
-        format(parseISO(declaration.informant.birthDate), 'dd MMMM yyyy'),
-        { ignoreCase: true }
-      )
+      ).toHaveText(formatDateTo_ddMMMMyyyy(declaration.informant.birthDate), {
+        ignoreCase: true
+      })
```

Comment:
Is the case something we need to ignore? Would be nice if the formatting was all we needed